vowpalwabbit/automl.cc

```diff
@@ -494,16 +503,12 @@ void config_manager::update_champ()
     }
 
     // If challenger is worse ('worse function from Chacha')
-    if (configs[config_index].ips < champ_lower_bound)
-    {
-      // TODO: Update worse function, this seems way too strict (removes configs instantly)
-      // configs[config_index].state = Removed;
-    }
+    if (worse(configs[config_index], champ_config)) { configs[config_index].state = Removed; }
```

Comment:
should this be an else if?

Reply:
Both cases should never be true, but yes else if looks better